woocommerce/woocommerce-ios
# [Performance] Request only the required fields for order stats

Part of #9892
⚠️ Depends on #9953


## Description

When we load the My Store dashboard we sync the order stats. However, the response includes a number of fields we don't need. This PR updates the remote request in `OrderStatsRemoteV4` so we only request the fields we need for stats.

## Testing instructions

Confirm tests pass and the My Store stats load as expected when you build and run the app.

---
